docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,700 @@
+
+The only exception is if the following Android method/iOS object returns a status of `OPT_OUT`, which means that the DII has been opted out of UID2 and no identity/token should be generated for it:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().getIdentityStatus()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.identityStatus
+```
+
+</TabItem>
+</Tabs>
+
+The best way to determine if DII is required by the UID2 mobile SDKs is to always call the `getAdvertisingToken()` method when the app starts up or resumes:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().getAdvertisingToken()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.getAdvertisingToken()
+```
+
+</TabItem>
+</Tabs>
+
+If `getAdvertisingToken()` returns `null`, you'll need to generate a new token. Pass the DII into a call to the `generateIdentity()` method again.
+
+## Opt-Out Handling
+
+(**GWH note: reviewed this doc up to this point. Rest is partially unreviewed (and the below needs fixing).**)
+
+If the DII provided to the `generateIdentity()` method has been opted out of UID2, this method returns `null`. To check the status of the UID2, you can call the following method:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().getIdentityStatus()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.identityStatus
+```
+
+</TabItem>
+</Tabs>
+
+In this case you should avoid making repeated calls to check the identity, because if the DII has a status of opted out, the UID2 token is not generated. (**GWH__SW_12 is there any more to this than a wasted API call?**)
+
+## Enable Logging
+
+The UID2 mobile SDK can generate logs, which could help in debugging any issues during UID2 integration work. To enable logging, do the following:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+// During UID2Manager initialization:
+UID2Manager.init(
+  context = this,
+  isLoggingEnabled = true
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```swift
+UID2Settings.shared.isLoggingEnabled = true
+// On iOS, you must set UID2Settings before you first access UID2Manager.shared. Changes made to settings after first access are not read.
+```
+
+</TabItem>
+</Tabs>
+
+## Optional: UID2 GMA/IMA Plugin for GAM Secure Signal integration
+
+If you intend to generate UID2 tokens to send to Google GMA/IMA SDKs, assuming you have followed the instructions in the [Client-Side Integration Guide for Mobile](integration-mobile-client-side.md) or the [Client-Server Integration Guide for Mobile](integration-mobile-client-server.md), you must also add the UID2 GMA/IMA plugins into your mobile app. For instructions, refer to the applicable plug-in guide:
```

Comment:
i forgot to remove Client-Server Integration Guide for Mobile mentioned here (and vice versa in the client-server guide)

Reply:
Done, thx. I just said "assuming you've followed the instructions in this guide" and made it a shared piece for both docs.